File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/PopulationUSA.java
package com.kendoui.spring.models;

import java.util.ArrayList;
import java.util.List;

public class PopulationUSA {
    private String name;
    private Number value;
    private List<PopulationUSA> items = new ArrayList<PopulationUSA>();

    public PopulationUSA(String name, Number value) {
        setName(name);
        setValue(value);
    }
    
    public List<PopulationUSA> getItems() {
        return items;
    }
    
    public void setItems(List<PopulationUSA> items) {
        this.items = items;
    }
    
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Number getValue() {
        return value;
    }

    public void setValue(Number value) {
        this.value = value;
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/TreeMap/PopulationUSA.cs
using System;
using System.Collections.Generic;

namespace Kendo.Mvc.Examples.Models
{
    public class PopulationUSA
    {
        public PopulationUSA(string name, int value)
        {
            Name = name;
            Value = value;
            Items = new List<PopulationUSA>();
        }

        public string Name { get; set; }
        public int Value { get; set; }

        public List<PopulationUSA> Items { get; set; }
    }
}
